examples/randlanet_classification.py

```diff
@@ -0,0 +1,255 @@
+import os.path as osp
```

Comment:
Can we add a reference to the paper here, and expected output model performance if you have it?

Reply:
Added some reference to the paper. Do you mean expected performances based on my own experiment? Or by the authors ? (they have not evaluated their model on ModelNet nor ShapeNet)

Reply:
Thanks! I meant expected performance when running this script, but not super urgent if you don't have these numbers.